docs/source/tutorials/psij.ipynb

```diff
@@ -0,0 +1,547 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Launching Methods\n",
+    "To specify how the processes in your job should be started once resources have been allocated for it, pass the name of a launcher (e.g. `\"mpirun\"`, `\"srun\"`, etc.) like so: `JobSpec(..., launcher='srun')`."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Scheduling Information\n",
+    "To specify resource-manager-specific information, like queues/partitions, runtime, and so on, create a [JobAttributes](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_attributes.JobAttributes) and set it with `JobSpec(..., attributes=my_job_attributes)`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from psij import Job, JobExecutor, JobSpec, JobAttributes, ResourceSpecV1\n",
+    "\n",
+    "environment = 'local'\n",
+    "ex = JobExecutor.get_instance(environment)\n",
+    "\n",
+    "job = Job(\n",
+    "    JobSpec(\n",
+    "        executable=\"/bin/date\",\n",
+    "        resources=ResourceSpecV1(node_count=1),\n",
+    "        attributes=JobAttributes(\n",
+    "            queue_name=\"<QUEUE_NAME>\", project_name=\"<ALLOCATION>\"\n",
+    "        ),\n",
+    "    )\n",
+    ")\n",
+    "\n",
+    "print(str(job.status))\n",
+    "ex.submit(job)\n",
+    "job.wait()\n",
+    "print(str(job.status))\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: The <QUEUE_NAME> and <ALLOCATION> fields will depend on the system you are running on."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Managing Job State\n",
+    "In all the above examples, we have submitted jobs without checking on what happened to them. Once the job has finished executing (which, for */bin/date*, should be almost as soon as the job starts) the resource manager will mark the job as complete, triggering PSI/J to do the same via the [JobStatus](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_status.JobStatus) attribute of the job. `Job` state progressions follow this state model:\n",
+    "\n",
+    "<img src=\"./images/psij_job_states.png\" width=\"350\"/>\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Cancelling a Job\n",
+    "If supported by the underlying job scheduler, PSI/J jobs can be canceled by invoking the [cancel](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job.cancel) method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = Job(JobSpec(executable=\"/bin/date\"))\n",
+    "ex.submit(job)\n",
+    "job.cancel()\n",
+    "print(str(job.status))"
```

Comment:
maaaybeee. I don't think `job.cancel()` is guaranteed to be synchronous. Nor can you guarantee that the call to `cancel()` is serviced before the job completes or fails. So `job.status` right after `cancel()` may show many things.

Reply:
I'll add a note to make it clear that it could possibly fail (i.e. not cancel in time)

Reply:
This comment has been addressed